ci/verify-check-cfg/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+# NOTE: Cannot be in workspace because of rustc 1.0 support
+name = "autocfg-verify-check-cfg"
+description = "A dummy crate to verify autocfg is emitting check-cfg directives"
+version = "0.1.0"
+edition = "2021"
```

Comment:
I think this test could work even on ancient Rust (`edition = "2015"`). At a glance, it will just need to add `extern crate autocfg;` and change to `std::u32::MAX`.

Reply:
Fixed with commit 027f73a911b748aecf70b0fc9f904fcf670f5879